app/src/org/commcare/android/logging/ReportingUtils.java

```diff
@@ -104,4 +104,15 @@ public static String getCommCareVersionString() {
         }
     }
 
+    public static int getAppVersion() {
+        return CommCareApplication.instance().getCurrentApp().getAppRecord().getVersionNumber();
```

Comment:
I think these will crash if there isn't an app successfully seated yet. Usually good to wrap these reporting helpers in a `try`/`catch` so that we guarantee we get the report

Reply:
Also any reason not to use `getVersion` above? Think it gives more diagnostic information

Reply:
Yeah will wrap it in try catch. And from the `getCommCareVersionString` we get an output like `CommCare Android, version \"2.38.1\"(439996). App v3490. CommCare Version 2.38. Build 439996, built on: 2017-08-25` .  Crashlytics automatically adds the Commcare version so All I wanted to get is ccz version to make errors searchable on that. Other information doesn't really seem useful.

Reply:
(yes)